### DISCUSSION TITLE
We couldn't send statistics to Appery.io. Please make sure that it's correct!

### DISCUSSION THREAD

**Discussion by yang1158280920** [exadel-inc/CompreFace#1038] [Q&A]
![33fba9528e40adbda7d9b3c1fa9ecf1](https://user-images.githubusercontent.com/61954441/223100692-e7af9a97-371e-4a70-b22d-ff49e4590f27.jpg)

**Comment by yang1158280920** [comment]
Problems with offline account registration

**Reply by pospielov** [reply]
This is definitely a bug. It should be a warning, not an error, and UI should continue working.
If you want to use CompreFace without the internet, I can suggest not accepting sending statistics at all:
![image](https://user-images.githubusercontent.com/3736126/223223616-d8519a10-f2a7-4b25-8e48-df9f0a829363.png)

In your case, as I understand, you already accepted it.
You can delete your DB so that you can register a new account without statistics. 
To do this, you need:
1. Run `docker volume ls`
2. Find volume with name <folder>_postgres-data, e.g. compreface_110_postgres-data.
3. Delete it with command:  `docker volume rm compreface_110_postgres-data`

**Reply by yang1158280920** [reply]
![image](https://user-images.githubusercontent.com/61954441/223287477-c1691bf9-26d5-4e85-aa20-208e91ddd539.png)
My registration page is like this, not checked

**Reply by pospielov** [reply]
You need to make this steps first:
1. Run `docker volume ls`
2. Find volume with name _postgres-data, e.g. compreface_110_postgres-data.
3. Delete it with the command: `docker volume rm compreface_110_postgres-data`
If you did those steps but it still doesn't show the checkbox, try to open it in a private window. But make sure when you start CompreFace, there is no volume
